terraform-frontend/terragrunt.hcl

```diff
@@ -43,5 +43,16 @@ provider "aws" {
     role_arn = "arn:aws:iam::$${var.target_aws_account_id}:role/BCGOV_$${var.target_env}_Automation_Admin_Role"
   }
 }
+
+# Additional provider configuration for us-east-1 region; resources can reference this as `aws.east`.
+# This is essential for adding WAF ACL rules as they are only available at us-east-1.
```

Comment:
should clarify that "for adding WAF to Cloudfront", us-east-1 is needed.

Reply:
I can add the reference link here if that helps. The reason it has to be in us-east-1 is also documented in this ticket as well.

Reply:
Added AWS reference link after this comment line..